word.js

```diff
@@ -0,0 +1,39 @@
+var Letter = require("./letter");
+
+var Word = function(value){
+	this.value = value;
+	this.letters = [];
+	this.numberGuesses = "";
+	for(var i=0; i<this.value.length; i++) {
+		this.letters.push(new Letter(this.value[i]));
+	}
```

Comment:
this could also be written like this using a higher order function:
```javascript
this.letters = this.value.map(function(item){
  return new Letter(item)
})
```
this can eliminate a couple lines and looks much cleaner.